.github/workflows/get_release_notes.yml

```diff
@@ -0,0 +1,52 @@
+name: Get Release notes
+on: [workflow_dispatch]
+jobs:
+  release_notes:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fetch repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Get the list of allowed pull request labels
+        run: |
+          RED='\033[0;31m'
+          NC='\033[0m'
+          FILE_NAME=.pull-requests-allowed-labels-list
```

Comment:
How about we move this file to `.github` ?

Reply:
this in `.github/` folder already :-), this is a manual github work flow

Reply:
I mean `.pull-requests-allowed-labels-list` file

Reply:
It have nothing to do with SDK itself, so I see no point in keeping it in root 🤔